MaxText/configs/base.yml

```diff
@@ -116,3 +116,8 @@ adam_b1: 0.9 # Exponential decay rate to track the first moment of past gradient
 adam_b2: 0.95 # Exponential decay rate to track the second moment of past gradients.
 adam_eps: 1.e-8 # A small constant applied to denominator outside of the square root.
 adam_eps_root: 0. # A small constant applied to denominator inside the square root.
+
+# Stack trace parameters
+collect_stack_trace: True
+stack_trace_to_cloud: False  # display stack trace on console
```

Comment:
Nit: I think these comments should describe the variable, not the default value. I.e. for stack_trace_to_cloud I think "Uploads to cloud logging if true, else to the console if false"

stack_trace_interval_seconds is pretty self-explanatory (no need for comment) or could be "Stack trace collection frequency"